apps/omg/lib/omg/state/transaction/payment.ex

```diff
@@ -83,14 +74,8 @@ defmodule OMG.State.Transaction.Payment do
 
   def new(inputs, outputs, metadata)
       when Transaction.is_metadata(metadata) and length(inputs) <= @max_inputs and length(outputs) <= @max_outputs do
-    inputs =
-      inputs
-      |> Enum.map(fn {blknum, txindex, oindex} -> %{blknum: blknum, txindex: txindex, oindex: oindex} end)
-
-    outputs =
-      outputs
-      |> Enum.map(fn {owner, currency, amount} -> %{owner: owner, currency: currency, amount: amount} end)
-
+    inputs = Enum.map(inputs, &new_input/1)
```

Comment:
I think `@doc` for this function is no longer valid. It does not add zero inputs / outputs anymore.